src/Data/Geometry/Structure/Feature.hs

```diff
@@ -67,10 +73,22 @@ instance LikeJSArray "Object" FeatureCollection where
     {-# INLINE fromJSArray #-}
     fromJSArray = js_JSArrayToFC
 
+-- | JSON GeometryInput
+newtype GeometryInput = GeometryInput JSVal
+instance LikeJS "Object" GeometryInput where
+  asJSVal = js_deleteGiTimestamp
+
 ----------------------------------------------------------------------------------------------------
 -- Some Functions
 ----------------------------------------------------------------------------------------------------
 
+data ParsedGeometryInput x = ParsedGeometryInput
+  { pgiFeatureCollection :: FeatureCollection
+  , pgiErrors            :: JS.Array JSString
+  , pgiLatLonAlt         :: Maybe (Vector 3 x)
```

Comment:
Although I'm not totally sure after you have said this.

The thing is that the scenario service generates a custom SRID for every scenario we save in the DB. A new reference system for a scenario is metric. It is centered in a way that the origin (zero) is somewhere near the center of geometry and has corresponding lat/lon/alt in degrees. In this case it does make sense to keep lat/lon/alt and SRID in qua-view at the same time: we know it is metric, know its geolocation, and do not need to create a new SRID at the same time